bodhi/server/models.py

```diff
@@ -1951,8 +1951,11 @@ def greenwave_subject_json(self):
         """
         return json.dumps(self.greenwave_subject)
 
-    def update_test_gating_status(self):
-        """Query Greenwave about this update and set the test_gating_status as appropriate."""
+    def get_test_gating_info(self):
+        """Query Greenwave about this update and returned the information retrieved."""
```

Comment:
This method now has a return value and can raise a ```BodhiException```, so we should document both of these.

```
"""
Query Greenwave about this update.

Returns:
    dict: The response from Greenwave for this update.
Raises:
    BodhiException: When the ```greenwave_api_url``` is undefined in configuration.
"""
```

Reply:
Will do